src/karmen_backend/server/routes/users_me.py

```diff
@@ -70,7 +70,7 @@ def create_inactive_user():
             ).hexdigest(),
             activation_key_expires=activation_key_expires,
         )
-    send_mail.delay(
+    send_mail(
```

Comment:
This is wrong. `.delay` here doesn't mean it's delayed, it just means it's delegated to a celery worker which is the intended behavior. The proposed change actually means it will be processed by the flask webserver process. 

Therefore, this change needs to be reverted.
```suggestion
    send_mail.delay(
```

Reply:
Done.

Reply:
Done.